lib/SIL/IR/SILPrinter.cpp

```diff
@@ -1420,6 +1420,13 @@ class SILPrinter : public SILInstructionVisitor<SILPrinter> {
     }
   }
 
+  void printForwardingOwnershipKind(OwnershipForwardingMixin *inst,
+                                    SILValue op) {
+    if (inst->getForwardingOwnershipKind() != op.getOwnershipKind()) {
+      *this << " [" << inst->getForwardingOwnershipKind() << " forwarding] ";
```

Comment:
Can you use a syntax that is a postfix? That is:

```
unchecked_ref_cast %0 : $Foo, forwarding: @owned
```

I think with time this will make updating tests a lot easier.